front/SettingsForm.tsx

```diff
@@ -71,6 +71,7 @@ const SettingsForm = () => {
             <input
               className="form-control form-control-lg"
               type="text"
+              data-cy = "settings-username"
```

Comment:
Remove the space between 'data-cy' and '='. It should be written as 'data-cy="settings-username"'